spec/factories/groups.rb

```diff
@@ -7,7 +7,7 @@
       match_factory { :group_match }
     end
 
-    sequence(:number)
+    number { rand(10) }
```

Comment:
will it cause problems if there are multiple groups with the same number?
what's the problem with a sequence here?

Reply:
i wanted to change things :(